### ISSUE TITLE
Improve chat styles

### ISSUE THREAD

**Issue by hrishikeshpaul** [hrishikeshpaul/noq#61]
In the 'Contact' section of the chat, make the profile pictures available.

**Comment by RayZhang001** [comment]
After sending a message, the names in that message is showing the user's name

**Comment by hrishikeshpaul** [comment]
> After sending a message, the names in that message is showing the user's name

yep that is because the chat is not connected to the server yet! you can still refresh the page and try it out it'll work then

@RayZhang001
